docs/source/workflow/generate-types.mdx

```diff
@@ -201,5 +201,5 @@ Check out our example using Apollo Server with generated types on CodeSandbox:
     src="https://codesandbox.io/static/img/play-codesandbox.svg"
   />
 </a>
-<br>
+<br/>
```

Comment:
Ha! I pointed this out in a previous commit comment.

Reply:
Huh! I didn't know they didn't need to be closed, Netlify complained so I fixed it 😂

Reply:
Works for me. Self closed is more correct / less surprising to see IMO.